Source/GoBot/ViewDatabase.swift

```diff
@@ -263,6 +263,20 @@ class ViewDatabase {
         // connection.trace { print("\n\n\ntSQL: \($0)\n\n\n") }
     }
     
+    /// Deletes the database at the given path.
+    func dropDatabase(at path: String) async throws {
+        if isOpen() {
+            await close(andOptimize: false)
```

Comment:
This function closes all connections and, while doing that, removes the wal and shm files (as indicated in section 4 at https://www.sqlite.org/wal.html).

Below, `try FileManager.default.removeItem(atPath: filePath)` fails for those two files and the operation fails even though the three files are removed successfully.

I think the removeItem should not throw for wal and shm, or at least check if they exist before deleting.